spec/models/comment_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  context 'validations' do
+    it 'is valid with valid attributes' do
+      movie = Movie.create(title: 'Example Movie', release_date: Time.zone.today, director: 'Director Name',
+                           synopsis: 'Lorem ipsum')
+      user = User.create(first_name: 'John', last_name: 'Doe', user_name: 'johndoe', email: 'john.doe@example.com',
+                         phone_no: '1234567890', password: 'password')
+      review = Review.create(movie: movie, user: user, rating: 5, review: 'Great movie!')
+      comment = Comment.new(review: review, user: user, comment: 'I agree!')
```

Comment:
You can use FactorBot to create comment

Reply:
done apu, used factory bot to generate data for all the models